db/db.go

```diff
@@ -12,24 +12,16 @@ import (
 var DB *gorm.DB
 
 func init() {
+	var err error
 	name := os.Getenv("MYSQL_DATABASE")
 	host := os.Getenv("DB_HOST")
```

Comment:
Addressing the environment variables to a new variable, creates a memory pointer.
You can use them directly into the fmt.Sprintf